admin-dev/themes/new-theme/tsconfig.json

```diff
@@ -20,9 +21,8 @@
       "@scss/*": ["scss/*"],
       "@node_modules/*": ["node_modules/*"],
       "@PSVue/*": ["js/vue/*"],
-      "@PSTypes/*": ["js/types/*"]
+      "@PSTypes/*": ["js/@types/*"],
```

Comment:
Ah seems like the "namespace" (I'm not sure it's the right name for it) still exists, so why did you need to change it in all the files regardless?